README.md

```diff
@@ -2,16 +2,15 @@ CoinShuffle++
 =============
 
 [![Build Status](https://github.com/decred/cspp/workflows/Build%20and%20Test/badge.svg)](https://github.com/decred/cspp/actions)
-[![ISC License](https://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
```

Comment:
The license is at http://copyfree.org/content/standard/licenses/isc/license.txt.

However, I have no objections to it being removed either.